src/test/java/emissary/test/core/junit5/RegressionTest.java

```diff
@@ -147,19 +147,41 @@ protected String getInitialForm(final String resource) {
      * 
      * @return the XML element decoders.
      */
+    @Deprecated
     protected ElementDecoders getDecoders() {
         return IBaseDataObjectXmlCodecs.DEFAULT_ELEMENT_DECODERS;
     }
 
+    /**
+     * This method returns the XML element decoders.
+     * 
+     * @param resource the "resource" currently be tested.
+     * @return the XML element decoders.
+     */
+    protected ElementDecoders getDecoders(final String resource) {
+        return getDecoders();
```

Comment:
Don't reference the newly-deprecated method in the new overload, just give it the final implementation.
```suggestion
        return IBaseDataObjectXmlCodecs.DEFAULT_ELEMENT_DECODERS;
```

Reply:
Withdrawing this suggestion.  The call from here to `getDecoders()` is needed to insure that overrides of `getDecoders()` are invoked.